HIP_scientific_research.md

```diff
@@ -0,0 +1,121 @@
+
+- If as a result of this scientific computation, even at a very low level, electricity and bandwidth are consumed by the community, the following method can be used for the initial setup:
+- - a Participation in the distributed computing system starts actively for everyone, and then a method is created for shutting it off.
+     - a1- An award is given for the method and it is encouraged not to shut it off.
+     - a2- No award is given for the method.
+  - b- Participation in the distributed computing system starts inactively, and then a method is created for turning it on.
+    - b1- An award is given for the method and it is encouraged to turn on the system.
+    - b2- No award is given for the method.
+
+# Detailed Explanation
+
+**The scientific advancement that will result from this proposal is incredibly large compared to the individual cost of the energy spent. **
+
+We can see how diseases like COVID have affected humanity. It is not incorrect to say that the way vaccines were developed so quickly for this disease was through distributed computing clusters, which we refer to as supercomputers. By repeating millions of simulations with thousands of possibilities in minutes, the ability to find optimal calculations and molecules has made it easier to find vaccines and prevent the pandemic from lasting for decades and saving millions of lives. This is just one example of a current and simple use case that I can give.
+
+There can be various criteria for using this excess processing and memory power for scientific calculations and operations. For example, during transmission, these operations can be given a lower priority. This way, they do not affect the use of the helium network. And in this way, with the excess processing power, they can continue their calculations in a comfortable manner without affecting the necessary updates and system stability with a higher priority.
+
+Another system control mechanism can be memory and processor/system temperature controls. For example, the amount of memory used can be the amount calculated as free by the system. And it can be changed dynamically. This way, the functionality of the helium network will not be affected. The situation where the operation is stopped based on the processor temperature can also be considered as a method that does not affect the functionality of helium and allows scientific calculations to continue and prevent possible hardware failures.
+
+The algorithm is as follows:
+- The helium hotspot software should be run at a lower CPU priority.
+- If the amount of free memory is above a certain level, this amount of memory should be used.
+- If the system temperature rises to 70 degrees or above, the scientific calculation should be stopped..
+
+# Drawbacks
+
+- There are two reasons not to do this requires inter-agency coordination (berkeley etc, vs helium)
+- Difficulties in software development.
+
```

Comment:
Proof of Coverage is absolutely unaffected. because an application running at the bottom of the services required for Proof of Coverage will run. The linux operating system inside the hotspot will give priority to the helium hotspot applications no matter what.